module/solutions/helper.js

```diff
@@ -68,6 +70,53 @@ module.exports = class SolutionsHelper {
     });
   }
 
+   /**
+    * Update solution document.
+    * @method
+    * @name updateSolutionDocument
+    * @param {Object} query - query to find document
+    * @param {Object} updateObject - fields to update
+    * @returns {String} - message.
+    */
+
+   static updateSolutionDocument(query= {}, updateObject= {}) {
```

Comment:
@priyanka-TL  If we have already existing document in db then findOneAndUpdate will also help right ??